src/Domain/Model/Application.php

```diff
@@ -90,10 +90,6 @@ public function getType(): string {
 		return $this->type;
 	}
 
-	public function isActiveMembership(): bool {
```

Comment:
Theoretically, you can leave this in to avoid the comparison when assigning the template variables when calling the mailer and it makes the domain more expressive. But it's not important.